lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/entitlements.js

```diff
@@ -0,0 +1,17 @@
+import { entitlementActions } from '../actions/actionCreators';
+
+function entitlements(state = [], action){
+  switch(action.type){
+    case entitlementActions.fetch.SUCCESS:
+      //TO BE REMOVED when entitlements in store are displayed in table
+      console.log('Fetching entitlements success ', action.entitlements)
```

Comment:
Should these log statements be displayed to the user?

Reply:
so the entitlement success will be 'displayed to the user' in the form of the entire table in the page (in the follow on PR)

Reply:
In the failure case I could definitely do something more useful and display it to the user. I've got thoughts on How I would do that, i'll look into it a little.

Reply:
Let me rephrase - it doesn't seem evident to me why these `console.log` statements should be displayed in the browser console. 

Now I know that this is being merged into a feature branch and maybe these will eventually be removed in a future PR - that's great. I guess my point is if they're going to be removed anyways, why not remove them in this PR?